src/components/common.scss

```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Mono:500');
+@import url('https://fonts.googleapis.com/css?family=Roboto+Mono:100,300,400,500,700,900');
```

Comment:
Again, I don't think we need to load all these fonts. Please check Zeplin Styleguide

Reply:
@dadaphl We are loading [RobotoMono](https://fonts.google.com/specimen/Roboto+Mono) only. It's one font being loaded here as per the styleguide